src/domain.ts

```diff
@@ -12,7 +12,8 @@ export type CaseClassMember = {
 export type CaseClass = {
   name: string
   members: Array<CaseClassMember>
-  desc?: string
+  desc?: string,
+  isValueClass: boolean
```

Comment:
this is a new field from metarpheus 0.1.8, which detects this pattern

```scala
case class UserId(value: String) extends AnyVal
```

See https://docs.scala-lang.org/overviews/core/value-classes.html for more info.